main.py

```diff
@@ -93,6 +93,15 @@ def train(chunk_dir, save_file, load_file=None, generation_num=0,
                 logdir=logdir, num_steps=num_steps, verbosity=verbosity)
 
 
+def validate(holdout_dir, load_file=None, logdir=None, num_steps=1000):
+    tf_records = sorted(gfile.Glob(os.path.join(holdout_dir, '*.tfrecord.zz')))
+    preprocessing.SHUFFLE_BUFFER_SIZE = int(1e5)
```

Comment:
uh let's not do this :P

Reply:
i said this code wasn't actually ready for review, didn't i? :) :P

The other thing we can do, btw, is we can change the filter amount from 5% to ...something else. 50%?

Reply:
Do you suspect that I/O is now the bottleneck for training?

Reply:
On 9x9, probably.  I changed it because, when running validate standalone on past models, it'd take ~5 minutes to fill the shuffle buffer before doing anything.  And on validate, there's no need to shuffle at all.  I think it suggests that really the shuffle buffer size should be an argument on preprocessing.get_input_tensors